src/App/App.spec.js

```diff
@@ -3,9 +3,16 @@ import { MemoryRouter } from 'react-router-dom'
 import App from './App.js'
 
 // mock Panoptes auth calls
-import auth from 'panoptes-client/lib/auth'
-jest.mock('panoptes-client/lib/auth')
-auth.checkCurrent.mockResolvedValue(null)
+jest.mock('panoptes-client/lib/auth', () => {
+  return {
+    __esModule: true,
```

Comment:
I'm not sure if this is relevant but PJC's exports are CommonJS, not ESM.

Reply:
Here are the exports for the `auth` module. 

https://github.com/zooniverse/panoptes-javascript-client/blob/master/lib/auth.js#L341-L353

Reply:
👍 Thank you - I've simplified this, though I need to come back later and try using `nock`.